nni/retiarii/experiment/pytorch.py

```diff
@@ -144,6 +144,10 @@ def preprocess_model(base_model, trainer, applied_mutators, full_ir=True, dummy_
                            'do not use mutators when you use LayerChoice/InputChoice')
     if mutators is not None:
         applied_mutators = mutators
+
+    # Add mutations on evaluators
+    applied_mutators += process_evaluator_mutations(trainer, applied_mutators)
+
```

Comment:
I remember some mutators should be put at the end of applied_mutators. could you double check this logic?

Reply:
That's not relevant to this part. Those belong to model mutators. These are evaluator mutators.

And we don't support sharing choices between model and evaluator.